### PR TITLE
Fixes in Report Overview

### PR DESCRIPTION
**Pull request by madelondohmen**
### Changes
This PR makes the following changes to the Report Overview:
- Fix the breadcrumbs
- Change the page description text on the Scheduled Reports page
- Change the page description text on the Reports History page
- If scheduled_reports_table or reports_history_table is empty, show an empty state text (instead of an empty table)
- Remove the "Subreport Name" column in the Scheduled Reports table

### Demo
afbeelding
afbeelding
afbeelding

### QA notes
- Go to the Report page. 
- Check the two tabs "History" and "Scheduled". Check the intro text and the empty state text.
- Create a report without schedule and a scheduled report
- Check the two tabs again. The table should be visible (no empty state text) and counter should work

---

### Code Checklist

- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] I have checked the documentation and made changes where necessary.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

- [ ] Tickets have been created for newly discovered issues.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: rocky/reports/templates/report_overview/report_history_table.html [hunk 1]
```diff
@@ -2,141 +2,148 @@
...
+                        <td class="nowrap">{{ report.parent_report.observed_at|date }}</td>
+                        <td class="nowrap">{{ report.parent_report.date_generated }}</td>
+                        <td>
+                            {% if report.children_reports %}
+                                <button type="button"
+                                        class="expando-button icon ti-chevron-down"
+                                        data-icon-open-class="icon ti-chevron-down"
+                                        data-icon-close-class="icon ti-chevron-up"
+                                        data-close-label="{% translate "Close children report object details" %}"
+                                        aria-expanded="false"></button>
```
Comment (ppvg) [inline rocky/reports/templates/report_overview/report_history_table.html:65]: The open label is missing here:

```suggestion
                                        aria-expanded="false">
                                        {% translate "Open children report object details" %}
                                </button>
```

**Review by ammar92 — APPROVED** [review]

**Comment by noamblitz** [comment]
QA checklist for PR's
## Checklist for QA:

- [ ] I have checked out this branch, and successfully ran a fresh `make reset`.
- [ ] I confirmed that there are no unintended functional regressions in this branch:
  - [ ] I have managed to pass the onboarding flow
  - [ ] Objects and Findings are created properly
  - [ ] Tasks are created and completed properly
- [ ] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
- [ ] I checked the logs for errors and/or warnings and made issues where necessary

### What works:


- Message shown correctly when no reports available.
- Messages disappear when reports available.

### What doesn't work:

image

Styling incorrect

### Bug or feature?:

- _bullet point + screenshot (if useful) if it is unclear whether something is a bug or an intended feature._

**Comment by madelondohmen** [comment]
> ### What doesn't work:
> image
> 
> Styling incorrect

-------------------

@noamblitz Weird, this is what I see:
afbeelding

**Comment by noamblitz** [comment]
Its only when there are no reports.

**Comment by stephanie0x00** [comment]
## Checklist for QA:

- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
  - [x] I have managed to pass the onboarding flow
  - [x] Objects and Findings are created properly
  - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
- [x] I checked the logs for errors and/or warnings and made issues where necessary

### What works:
Looks good. The styling bug is either resolved, or a specific bug to the environment from @noamblitz. 

### What doesn't work:
n/a

### Bug or feature?:
Found #3734 but unrelated to this PR.

**Review by underdarknl — APPROVED** [review]

**Pull request merged**